metrics/internal/cache/pv.go

```diff
@@ -83,10 +86,14 @@ func NewPersistentVolumeStore(opts *options.Options) *PersistentVolumeStore {
 		monitorConfig:             cephMonitorConfig{},
 		cephClusterNamespace:      opts.AllowedNamespaces[0],
 		cephAuthNamespace:         opts.CephAuthNamespace,
+		CephFSPVList:              make(map[types.UID]string),
+		CephFSSbvolumeCountMap:    make(map[string]int),
+		FSName:                    "",
```

Comment:
how this works when the external mode and internal mode are using different FSNames?